test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Just as the content of the file reads, you should ignore this file. I noticed you ignored it in your `.gitignore` file but I believe it must have been cached before your stage your changes.

Please clear the cached `git rm -r --cached .` 
Stage your files and push them again to github.
Kindly verify that the file is not present in your repo before asking for another review.

Reply:
Thanks @mmsesay!
I have followed what you said and the file test.md has been ignored.